google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemTest.java

```diff
@@ -18,6 +18,8 @@
 
 import static com.google.common.truth.Truth.assertThat;
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
```

Comment:
You can use Truth.assertThat instead of Assert, As Truth, is already available in this module

Reply:
https://github.com/athakor/java-storage-nio/pull/2/files#diff-c124c6b76e7073c92e82eae0a0d124adL19